TDP/tdp-48.md

```diff
@@ -0,0 +1,130 @@
+The subDAO signer group will have discretionary power to disburse subDAO funds, provided that the funding use is in line with the proposed funding categories and within the total budgeted amount for the period, including any buffer amount.
+
+Any spending for uses outside the proposed categories would require an additional governance process.
+
+### Proposed budget
+### mUSD
+The total amount of mUSD requested under this funding request is $196,614 per month or $786,456 for 4 months.
+
+The current value of non-MTA assets held by the TreasuryDAO & subDAOs is around $3.4 millon. When considering likely costs across other areas such as the Ecosystem subDAO, protocol costs and signer costs, and considering current protocol revenue, this gives the project a runway of around 13 months based on non-MTA assets. This runway length assumes maximum spend by the Builder subDAO, including the buffer amount described below, and therefore reflects the ‘worst case’ rather than intended spend.
+
+It is proposed that $196,614 is transferred upon approval of this funding request, with a [LlamaPay](https://llamapay.io/) stream set up for the remaining amount, delivered between the earliest practical start date and the 30th of November, 2022.
+
+This will provide an upfront balance to be used immediately, while ensuring that the overall amount custodied by the subDAO is no larger than necessary.
+
+The mStableDAO is expected to hold the subDAO accountable for using the funds as outlined and for meeting deliverables over the funding period as defined in this proposal.
+
+The breakdown of this proposed budget is below. Note that these are estimates and it is expected that there will be variance in the exact distribution of funds across each month and across each category.
+
+![mUSD Breakdown](../assets/TDP-48/musd-breakdown.png)
+
+A 10% buffer has been added to allow flexibility and ensure that smaller unexpected costs can be met.
+
+Where salaries are denominated in Euro, an exchange rate of 0.96 USD/EUR has been used for conversion to allow small amount of room for movement in the rate.
+
+All unallocated budget after 4 months will be returned to the TreasuryDAO or, at the discretion of the TreasuryDAO, may be subtracted from any subsequent approved funding request. Note that any calculation of unallocated budget should take into account any outstanding accounts payable to ensure that the subDAO can meet future obligations.
+
+To ensure that the subDAO is not incentivized to overspend and use the buffer amount, the TreasuryDAO should hold the Builder subDAO accountable to the original budget and consider use of the buffer as a negative indicator of performance.
+
+### MTA
+In addition to the proposed funding in USD stablecoins, a total amount of 1,175,000 MTA is requested to allow the Builder subDAO to handle the MTA allocations for current team members as per their current agreements with the mStableDAO.
+
+![MTA Breakdown](../assets/TDP-48/mta-breakdown.png)

```

Comment:
this looks like a table, you can add tables in markdown

Reply:
Fixed